File: sms-api-java/sendmessageservice/src/main/java/com/example/sendmessageservice/controller/dto/AuthDto.java
package com.example.sendmessageservice.controller.dto;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Data
@Getter
@Setter
public class AuthDto {

    private String token;

    public AuthDto() {
    }

    public AuthDto(String token) {
        this.token = token;
    }
}

File: sms-api-kotlin/sendmessageservice/src/main/kotlin/com/example/sendmessageservice/controller/dto/AuthDto.kt
package com.example.sendmessageservice.controller.dto

import lombok.Data
import lombok.Getter
import lombok.Setter


@Data
@Getter
@Setter
class AuthDto {
    private var token: String? = null

    constructor()
    constructor(token: String?) {
        this.token = token
    }
}
